lib/BuildSystem/BuildNode.cpp

```diff
@@ -27,10 +27,10 @@ bool BuildNode::configureAttribute(const ConfigureContext& ctx, StringRef name,
                                    StringRef value) {
   if (name == "is-directory") {
```

Comment:
Maybe at the same time we are doing this we should introduce a new setter ("type"), and soft deprecate these old ones?